webpack.config.js

```diff
@@ -61,16 +63,27 @@ module.exports = [{
           presets: ['es2015', 'react'],
         },
       },
+
+      /*
+       * Parse SCSS to minified CSS, then postprocess with postcss autoprefixer plugin
+       */
       {
         test: /\.scss$/,
-        loader: ExtractTextPlugin.extract(['css','sass']),
+        loader: ExtractTextPlugin.extract('style', ['css?minimize', 'postcss', 'sass']),
```

Comment:
I'm wondering what order these run in. Is the current CSS minified? maybe we could switch the order around so it ends up minified.